Makefile

```diff
@@ -3,3 +3,10 @@ docker := docker run -e -t -i --sig-proxy=true --rm -v $(shell pwd):/foo -w /foo
 
 lint:
 	@$(docker) prow/community-lint.sh
+
+test:
+	go test ./...
+
+sync:
```

Comment:
I am not 100% certain if the `go get` will work properly given the current state of `k8s.io/test-infra` `go.mod` (https://github.com/kubernetes/test-infra/issues/14070). If it doesn't we can fallback to using the docker image `gcr.io/k8s-prow/peribolos` (or clone repo).

Reply:
Yep, looks like it fails. Lets go with the docker image approach, seems the simplest. There is no make though, I'll just inline the command in test-infra

Reply:
I removed this and update test-infra